lib/subscriber.js

```diff
@@ -49,6 +49,11 @@ class SubscriptionContext {
     return this.pubsub.subscribe(topic, this.queue).then(() => this.queue)
   }
 
+  asyncIterator (topics) {
```

Comment:
why is this called `asyncIterator`? Can't we just support arrays of topics in `subscribe`?

Reply:
I borrowed the naming of graphql-subscriptions. As far as I know, we should return asyncIterator to subscribe of GraphQLObjectType, and this is a naming meaning of custom asyncIterator.

However, as I try to add additional features like withFilter, it is not good to implement in the current PubSub and SubscriptionContext separate structure. So, as you say, it would be better to add only support arrays to existing subscribe function.